Tavac75/Pawfect-Project
# Home

I have a home page built. It is rather simple. it leads to the login page.

Testing steps:
1. At the login page you can either login or go to the create user page.
2. Try to login. You have not made a user account. You should not be able to login.
3. Go to the create user page, and create a user. Your session will be started upon acceptance. That takes you to the search page. There is nothing to see there it is just a place holder, but you can logout, and that takes you back to the login page.
4. Now that you have an account you can login. You should have a session key. This once again takes you to the search page.
5. Log out. 

If you have any questions or concerns hit me up on slack. Thanks!
